samples/bluetooth/peripheral_gatt_dm/sample.yaml

```diff
@@ -5,6 +5,6 @@ tests:
   samples.bluetooth.peripheral_gatt_dm:
     build_only: true
     build_on_all: true
-    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf52dk_nrf52810
+    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840
```

Comment:
Did you check the compilation with nrf51dk_nrf51422? Should this be marked as supported in the Readme?